src/SavedSearches/WriteModel/SavedSearchRepositoryInterface.php

```diff
@@ -20,10 +21,12 @@ public function update(
         string $userId,
         string $name,
         QueryString $queryString
-    ): int;
+    ): void;
 
     public function delete(
         string $userId,
         string $searchId
     ): void;
+
+    public function findSavedSearchOwnedByCurrentUser(string $id): ?SavedSearch;
```

Comment:
This interface is confusing me, all the other interfaces pass in an `id` and a `userId`. I think it's best to stick with that approach and change the method to `findByIdAndUserId`.

I would also remove the `userId` property from the constructor of `UDB3SavedSearchRepository`

Reply:
I now see that this new method is inside the wrong interface, there is a read and a write interface. This method should be part of `CultuurNet\UDB3\SavedSearches\ReadModel\SavedSearchRepositoryInterface`

Reply:
Moved it to a separate SavedSearchReadRepository class.